Classes/Lib/Searchresult.php

```diff
@@ -312,6 +312,6 @@ public function buildTeaserContent(string $content): string
             }
             return $teaser;
         }
-        return $cObj->crop($content, $this->conf['resultChars'] . '|…|1');
+        return $cObj->crop($content, $this->conf['resultChars'] ?? 0 . '|…|1');
```

Comment:
This change seems to have broken the ... cropping from the results. It lacks some braces 
`($this->conf['resultChars'] ?? 0) . '|…|1') `